packages/babel-helpers/src/helpers/applyDecs2301.js

```diff
@@ -75,35 +81,37 @@ function memberDec(
   var get, set, has;
   if (isPrivate) has = privateHas;
   else {
-    has = function (receiver) {
-      return name in receiver;
+    has = function (target) {
+      return name in target;
     };
   }
   if (!isPrivate && (kind === 0 /* FIELD */ || kind === 2) /* METHOD */) {
-    get = function (receiver) {
-      return receiver[name];
+    get = function (target) {
+      return target[name];
     };
     if (kind === 0 /* FIELD */) {
-      set = function (receiver, v) {
-        receiver[name] = v;
+      set = function (target, v) {
+        target[name] = v;
       };
     }
   } else if (kind === 2 /* METHOD */) {
     // Assert: isPrivate is true.
-    get = function () {
-      // TODO: Throw if !Has(receiver, name).
+    get = function (target) {
+      assertInstanceIfPrivate(privateHas, target);
       return desc.value;
     };
   } else {
     // Assert: If kind === 0, then isPrivate is true.
     var t = kind === 0 /* FIELD */ || kind === 1; /* ACCESSOR */
     if (t || kind === 3 /* GETTER */) {
       get = function (target) {
```

Comment:
We can generate a `get` with `assertInstanceIfPrivate` if `isPrivate`, so multiple `context.access.get` call does not require multiple `if(has)` check.

Reply:
I mean:

```js
if (isPrivate) {
  get = function (target) {
     desc.get.call(assertInstanceIfPrivate(hasPrivateBrand, target));
  }
} else {
  get = function (target) {
     desc.get.call(target);
  }
}
```
because `isPrivate` is known at compile time, the check should not be duplicated in `context.access.get`.